ajax/libs/filter.js/1.5.2/filter.js

```diff
@@ -0,0 +1,1727 @@
+/*
+ * filter.js
+ * 2.1.0 (2016-08-09)
```

Comment:
Why `2.1.0` here? Are you sure about what you're doing?

https://github.com/jiren/filter.js/blob/v1.5.2/filter.js#L3